console/console-init/ui/src/modules/project/utils/ProjectFormatter.tsx

```diff
@@ -68,13 +68,13 @@ const getStatusIconByPhase = (phase: string) => {
     </span>
   );
 };
-const formatString = (string?: string) => {
-  if (string) {
-    const formattedString =
-      string[0].toUpperCase() + "" + string.substring(1, string.length);
-    return formattedString;
-  }
-};
+// const formatString = (string?: string) => {
```

Comment:
please delete commented code if not in use.

Reply:
As discussed earlier, these block of codes were unused and and were giving warnings. I had asked everyone to let me know if these were created for future use. Which is why I chose to comment this instead of removing it straight away.